pkg/docker/docker_util.go

```diff
@@ -30,7 +30,7 @@ var (
 	ImageName               = "cr.flyte.org/flyteorg/flyte-sandbox:dind"
 	FlyteSandboxClusterName = "flyte-sandbox"
 	Environment             = []string{"SANDBOX=1", "KUBERNETES_API_PORT=30086", "FLYTE_HOST=localhost:30081", "FLYTE_AWS_ENDPOINT=http://localhost:30084"}
-	FlyteSnackDir           = "/usr/src"
+	FlyteSnackDir           = "/root"
```

Comment:
Can we rename this to source to keep in sync with the flag

Reply:
ok